src/Support/helpers.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+if (! function_exists('array_flatten')) {
+    /**
+     * This code snippet is inspired by Laravel.
+     *
+     * @see https://github.com/laravel/framework/blob/ecbc4622021b10287c36757a2867ba343f7698a0/src/Illuminate/Collections/Arr.php#L211-L232
+     */
+    function array_flatten(iterable $iter, float $depth = INF): array
```

Comment:
Instead of adding a function to the global namespace, create an namespaced class `Array` with a static method `flatten`.

Reply:
Also add tests for this new function.